src/app/components/icons/utilityFunctions.tsx

```diff
@@ -15,21 +17,58 @@ export const getMuiIconName = (filename: string): string => {
     return muiName;
 };
 
+export const changeSvgColorAndSize = (svg: string, color: string | undefined, size: number | undefined): string => {
+    let newSvg = svg;
+    
+    if (color) {
+        // @ts-ignore
+        const colorCode = color === 'White' ? Colors[color.toLowerCase()][50] : Colors[color.toLowerCase()][500];
```

Comment:
You can skip this color lookup logic if we use the literal color values in the Select (see comment above). Or you could pull this out to another function that will be easier to extend in the future if we want to do more colors:

```
const getColorCode = (color:string) => {
    const lowerColor = color.toLowerCase();
    switch(lowerColor){
        case 'black':
        case: 'gray':
        case: 'blue':
            return Colors[lowerColor as (keyof typeof Colors)][500];
        case 'white':
            return Colors.white[50];
        default:
            return Colors.black[500];
    }
}
```